### PR TITLE
RUN-5477: Updated electron.d.ts and modules.d.ts

### PR DESCRIPTION
**Pull request by daniel-koc**
#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
- [ ] manual tests are [changed or added](https://github.com/openfin/test_apps)
- [ ] relevant documentation is [changed or added](https://github.com/hadoukenio/js-adapter)
- [ ] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [ ] PR release notes describe the change in a way relevant to app-developers
- [ ] Link to new tests added
- [ ] PR has assigned reviewers

#### Release Notes

Notes:

### CODE REVIEW

File: src/modules.d.ts [hunk 1]
```diff
@@ -33,13 +33,13 @@ declare namespace Electron {
         fromProcessAndFrameIds: (processId: number, frameId: number) => WebContents;
         getOwnerBrowserWindow: () => BrowserWindow | void;
         mainFrameRoutingId: number;
-        session: Session;
+        readonly session: Session;
```
Comment (pbaize) [inline src/modules.d.ts]: Let's delete this line and use the definition from electron.d.ts (just built locally and that's fine)

File: src/modules.d.ts [hunk 2]
```diff
@@ -33,13 +33,13 @@ declare namespace Electron {
...
         registerIframe: (frameName: string, frameRoutingId: number) => void;
         unregisterIframe: (frameName: string, frameRoutingId: number) => void;
     }
 
     export interface BrowserWindow {
-        id: number;
+        readonly id: number;
```
Comment (pbaize) [inline src/modules.d.ts]: Let's delete this line and use the definition from electron.d.ts (just built locally and that's fine)

File: src/modules.d.ts (hunk 1 above)
Comment (daniel-koc) [reply src/modules.d.ts]: done

File: src/modules.d.ts (hunk 2 above)
Comment (daniel-koc) [reply src/modules.d.ts]: done

**Comment by daniel-koc** [comment]
@pbaize  Merge this PR please if you think it's OK, I don't have permission to do merges for core

**Pull request merged**